analytical/templatetags/piwik.py

```diff
@@ -64,9 +81,22 @@ def __init__(self):
                                  "must be a (string containing a) number")
 
     def render(self, context):
+        custom_variables = context.get('piwik_vars', ())
+
+        complete_variables = (var if len(var) >= 4 else var + (DEFAULT_SCOPE,) for var in custom_variables)
```

Comment:
The outer parenthesis are redundant.

Have you run `flake8`? (Doesn't it complain?)